services-custom/dynamodb-enhanced/src/main/java/software/amazon/awssdk/enhanced/dynamodb/internal/converter/attribute/LocalDateTimeAttributeConverter.java

```diff
@@ -35,10 +35,13 @@
  * A converter between {@link LocalDateTime} and {@link AttributeValue}.
  *
  * <p>
- * This stores and reads values in DynamoDB as a number, so that they can be sorted numerically as part of a sort key.
+ * This stores and reads values in DynamoDB as a string.
  *
  * <p>
- * LocalDateTimes are stored in the format "[-]YYYYMMDDHHIISS[.NNNNNNNNN]", where:
+ * Values are stored in ISO-8601 format, with nanosecond precision and a time zone of UTC.
```

Comment:
We should not store timezone information for LocalDateTime. The actual string format being proposed omits it, but this description implies it will be there (the 'Z')